custodian/vasp/handlers.py

```diff
@@ -66,17 +67,20 @@ class VaspErrorHandler(ErrorHandler):
     """
 
     is_monitor = True
+    max_natoms: int = 50
```

Comment:
It's likely worth adding documentation here because it's not immediately obvious to the end-user what this is being used for.

With numpy docstrings, you can do

```
Attributes
-----------
is_monitor
    blah blah
max_natoms
    blah blah blah
```

Also, I think `max_natoms` might be misleading because the user might think that they should (for some reason) be specifying the maximum number of atoms across all the systems they intend to run with Custodian when in reality, it is simply a tolerance used elsewhere. Admittedly, I am not sure of a better name (maybe `natoms_tol` (meh...).

Reply:
Fleshed out the docstr a bit and renamed this to `num_sites_kpoint_cutoff`. Not the most appealing name but hopefully clearer